tests/utils/tools.php

```diff
@@ -391,10 +391,6 @@ function fromJSON($var) {
     return $func($var);
 }
 
-
-
-/* NOTE: Using this function will take down mongod ! */
```

Comment:
Is this no longer true?
It used to quite literally (and intentionally) result in an abort() to make sure the getmore failed completely